docs/06-contributors/020-development.md

```diff
@@ -169,8 +169,9 @@ Note: In CI, tests likely run much slower than on your local machine, so you may
 
 ## How do I work only on the compiler?
 
-The following command runs the cargo tests, currently just ensures the valid examples compile and the
-invalid ones do not.
+The following command runs the rust tests in wingc, including verification that valid tests compile, invalid tests do not compile, and none of them panic.
+
+It will also make sure to update any snapshots.
```

Comment:
Does this VS code thing work with our tests? https://github.com/mitsuhiko/insta#editor-support (see gif) Looks like it could be handy, maybe we can add it to the contributing guide.

Reply:
Good point, I'll add a couple of notes about that as well as the fully optional insta cli